plugins/BEdita/Core/src/Model/Behavior/UniqueNameBehavior.php

```diff
@@ -116,9 +133,27 @@ public function uniqueNameExists($uname, $id = null)
      * @param \Cake\Event\Event $event The event dispatched
      * @param \Cake\Datasource\EntityInterface $entity The entity to save
      * @return void
+     * @codeCoverageIgnore
```

Comment:
I'd rather keep this line uncovered, even if it's quite trivial. This is not a method that simply adds configuration like `Table::initialize()`…